service/db/src/test/java/gov/va/vro/service/db/mapper/ClaimInfoResponseMapperTest.java

```diff
@@ -43,9 +43,11 @@ void testMain() {
     ContentionEntity contentionEntity = new ContentionEntity();
     contentionEntity.setDiagnosticCode(diagnosticCode);
 
+    final Boolean sufficientEvidence = true;
     AssessmentResultEntity assessmentResultEntity = new AssessmentResultEntity();
     Map<String, String> areEvidenceInfo = ImmutableMap.of("k1", "v1", "k2", "v2");
     assessmentResultEntity.setEvidenceCountSummary(areEvidenceInfo);
+    assessmentResultEntity.setSufficientEvidenceFlag(sufficientEvidence);
```

Comment:
This is good except it looks to me that we are not setting it in the real code; i.e SaveDbServiceImpl.insertAssessmentResult. So your end to end test will always pass since it is based on setSufficientEvidenceFlag.

Reply:
We do set it in real code. We simply do not set it on immediate creation like the mapper test does, but it's trying to test the mappings so I don't see any need to mimic an e2e process here.

As for where this update happens:

There is a method in SavetoDbServiceImpl.java with this signature:
  public void updateSufficientEvidenceFlag(AbdEvidenceWithSummary evidence, String diagnosticCode) {

Which is called by MasAssessmentResultProcessor

Now why we make 2 calls instead of all at once I do not know, possibly to not break other paths in v1.

Reply:
it looks like there's only one use of both save to dbserviceimpl calls so i should just collapse them into one method however.